### PR TITLE
Add metrics from multiple API endpoints

### PR DESCRIPTION
**Pull request by therve**
### What does this PR do?

Allows to scrape multiple API endpoints exposed by EventStore for metrics, including:
- statistics for projections
- statistics for subscriptions
- statistics for clustering (gossip) when configured

### Motivation

Production systems may require monitoring for projections to ensure their consistency, as well as gain insight into the state of the cluster.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Most of the new metrics were modeled after the data exposed by [EventStore exporter](https://github.com/marcinbudny/eventstore_exporter/) for Prometheus.

Originally submitted at #509

### CODE REVIEW

**Comment by hithwen** [comment]
Ping @xorima for review

**Comment by Xorima** [comment]
@hithwen I am not actively maintaining this btw, so if you find anyone else who wants to maintain it feel free to pass it over to them, while I built this for work we decommissioned it before these metrics started getting used by us.

File: eventstore/datadog_checks/eventstore/eventstore.py [hunk 1]
```diff
@@ -63,12 +88,16 @@ def check(self, instance):
...
+                        tag_name, tag = tag.rsplit(':', 1)
+                        tag_path = self.get_tag_path(tag, path, eventstore_paths)
+                    else:
+                        tag_path = self.get_tag_path(tag, path, eventstore_paths)
+                        tag_name = self.format_tag(tag_path.split('.')[-1])
                     tag_value = self.get_value(parsed_api, tag_path)
                     tag_builder.append('{}:{}'.format(tag_name, tag_value))
                 metric_builder['tag_by'] = tag_builder
-                metric_definitions[path] = metric_builder
+                metric_definitions[path].append(metric_builder)
```
[the remark this answers is not in the document]
Comment (hithwen) [reply eventstore/datadog_checks/eventstore/eventstore.py]: For example, in the integration test the `/info` endpoint has multiple entries for `state` and `projectionsModel`
image

File: eventstore/datadog_checks/eventstore/eventstore.py [hunk 2]
```diff
@@ -63,12 +88,16 @@ def check(self, instance):
                     tag_builder.append('instance:{}'.format(url))
                 tag_builder.append('name:{}'.format(name_tag))
                 for tag in tags:
-                    tag_path = self.get_tag_path(tag, path, eventstore_paths)
-                    tag_name = self.format_tag(tag_path.split('.')[-1])
+                    if ':' in tag:
```
[the remark this answers is not in the document]
Comment (hithwen) [reply eventstore/datadog_checks/eventstore/eventstore.py]: Is just some enpoints send the tags in that format. Included example as a comment

**Pull request merged**